site/docs/voice/webhooks/realtimeTranscriptionRejected.mdx

```diff
@@ -0,0 +1,148 @@
+---
+id: realtimeTranscriptionRejected
+title: Real-Time Transcription Rejected
+slug: /voice/webhooks/realtimeTranscriptionRejected
+description: A general overview of Bandwidth's Real-Time Transcription Rejected Webhook
+keywords:
+  - bandwidth
+  - voice
+  - webhook
+  - transcribing
+  - rejected
+hide_title: false
+image: ../../static/img/bandwidth-logo.png
+---
+
+This event may be sent to the url specified when sending a [`<StartTranscription>`][1] verb. This event will be sent if the real-time transcription requested in the `<StartTranscription>` verb was not started.
+
+## Request Parameters
+TODO `s/originalTranscription/originalRealtimeTranscription`?
```

Comment:
s?

Reply:
This just means "replace `originalTranscription` with `originalRealtimeTranscription`?"